impl/src/templates/remote_crates.bzl.template

```diff
@@ -17,8 +17,8 @@ def _new_git_repository(name, **kwargs):
 
 def {{workspace.gen_workspace_prefix}}_fetch_remote_crates():
 {%- if crates %}
-    {% for crate in crates %}
-        {%- if crate.source_details.git_data %}
+{% for crate in crates %}
+    {%- if crate.source_details.git_data %}
```

Comment:
```suggestion
{%   for crate in crates %}
    {%-   if crate.source_details.git_data %}
```

Reply:
the actual fix we probably want here is just to make the block for the for loop `{%-` instead of `{%` so that the whitespace is removed https://github.com/Keats/tera/blob/master/docs/content/docs/_index.md#whitespace-control

Reply:
@UebelAndre for now I'm just going to revert the changes related to this since I don't want to block this PR on that. Could you open up a separate PR with these changes?

Reply:
Yeah, I can do that. Thanks for the attempt 😄

Reply:
I created #207